_data/paths/accessible.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  'basePath': 'content/en/accessible',
+  'owners': [],
```

Comment:
Was this meant for code owners? If so, we may be able to rely on [GitHub CODEOWNERS](https://help.github.com/articles/about-code-owners/) instead. I've [already started using it a bit on the site](https://github.com/GoogleChrome/web.dev/blob/master/CODEOWNERS).

Reply:
Sure, that works.